docs/dml-of-carbondata.md

```diff
@@ -402,6 +404,11 @@ CarbonData DML statements are documented here,which includes:
 
 ## UPDATE AND DELETE
 
+  Since the data in CarbonData files is immutable, the updates and delete are done via maintaining two files namely:
```

Comment:
Can update the line based on filesystem. Like, ` Since the data stored in file system like HDFS is immutable,..`